src/_zkapauthorizer/tests/test_storage_server.py

```diff
@@ -139,6 +139,25 @@ def test_raise_for(self):
             )
 
 
+def read_count(storage_server):
```

Comment:
I gave them more precise names, hopefully that also ends up being more descriptive.  They also have docstrings now, in any case, which I guess should help.  I left them as accepting a storage server because I like having the metric implementation details concentrated in one place instead of spread across all of the test methods.